### ISSUE TITLE
Include table 5 reproduction as the regression test

### ISSUE THREAD

**Issue by tylergu** [xlab-uiuc/acto#251]
The 56 bugs reproduction suite is pushed to the sosp-ae branch, and we are able to reproduce all of them.
We should include this suite as part of the regression test for PRs.
However, the entire reproduction process takes ~15 hours to run, and I am not sure the github action machines are suitable for running it. We probably need to register one of the lab machine to run this CI.

Todo after the AE deadline:
- [x] Change every bug from table 5 to a test, and include all 56 bugs as part of the regression test
- [x] Configure to run this regression test on a lab machine
- [x] Nice to have: parallelize the test running

**Comment by tianyin** [comment]
@tylergu I assume that each bug can be reproduced by one test? 

With your number, a test takes on average 16 minutes to run? That does not match what is claimed in the paper.

Can you elaborate why it runs so slowly?

**Comment by tylergu** [comment]
@tianyin Because we do not run they in parallel for the reproduction
